tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -231,6 +235,26 @@ pub mod pallet {
 				Ok(())
 			})
 		}
+
+		#[cfg(not(feature = "production"))]
+		#[pallet::call_index(5)]
+		#[pallet::weight({15_000_000})]
+		pub fn remove_identity(
+			origin: OriginFor<T>,
+			who: Identity,
+			identity: Identity,
+		) -> DispatchResult {
+			T::ManageOrigin::ensure_origin(origin)?;
+
+			ensure!(LinkedIdentities::<T>::contains_key(&identity), Error::<T>::IdentityNotExist);
+			ensure!(identity != who, Error::<T>::RemovePrimeIdentityDisallowed);
```

Comment:
It's for dev only, so I think we should allow the removal of prime identity.

We could even go one step further and change and function args (and CLI args) to:
```
  who: identity,
  identities: Option<Vec<Identity>>
```

So we remove a vector of identities in the `who`'s IDGraph, and if `identities` is `None`, we simply remove the whole IDGraph. How does that sound?

Reply:
It's also nice to support `remove [who]` directly -> remove the whole IDGraph.